examples/energy-management-app/esp32/main/Kconfig.projbuild

```diff
@@ -185,3 +185,50 @@ depends on ENABLE_PW_RPC
             about available pin numbers for UART.
 
 endmenu
+
+
+menu "Energy Management Example App Configuration"
+
+    config ENABLE_EXAMPLE_EVSE_DEVICE
+        bool "Enable EVSE device on EP1"
+        default "y"
+        help
+            To use Energy Management App as EVSE device and compile dependent files.
+
+    config ENABLE_EXAMPLE_WATER_HEATER_DEVICE
+        bool "Enable Water Heater device on EP1"
+        default "n"
+        help
+            To use Energy Management App as Water Heater device and compile dependent files.
+
+    config CHIP_DEVICE_CONFIG_ENABLE_DEVICE_ENERGY_MANAGEMENT_TRIGGER
+        bool "Enable DEM triggers"
+        default "y"
+        help
+            Enable test triggers used by the Device Energy Management tests
+
+    config CHIP_DEVICE_CONFIG_ENABLE_ENERGY_EVSE_TRIGGER
+        bool "Enable EVSE triggers"
+        default "n"
+        help
+            Enable test triggers used by the Evse tests
+
+    config CHIP_DEVICE_CONFIG_ENABLE_WATER_HEATER_MANAGEMENT_TRIGGER
+        bool "Enable Water Heater triggers"
+        default "n"
+        help
+            Enable test triggers used by the Water Heater Management tests
+
+    config DEM_SUPPORT_STATE_FORECAST_REPORTING
+        bool "State forecast reporting is supported"
+        default "n"
+        help
+            Ensure kStateForecastReporting (SFR) is set in the DeviceEnergyManagement::Feature attribute. Note cannot set SFR and PFR.
```

Comment:
There may be a KConfig mechanism to make these 2 mutually exclusive.
https://github.com/torvalds/linux/blob/v4.15/Documentation/kbuild/kconfig-language.txt#L104